hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneListStatusHelper.java

```diff
@@ -227,6 +208,36 @@ public Collection<OzoneFileStatus> listStatusFSO(OmKeyArgs args,
         Collectors.toList());
   }
 
+  private OzoneFileStatus getStatus(String prefixPath, long scmBlockSz,
```

Comment:
nit: indentation is wrong. I guess check-style will fail.
```suggestion
  private OzoneFileStatus getStatus(String prefixPath, long scmBlockSz,
                                    String volumeName, String bucketName,
                                    ReplicationConfig bucketReplication,
                                    ListIterator.HeapEntry entry) {
```

Reply:
done